java-agent/agent-tooling/src/main/java/io/opentelemetry/auto/agent/tooling/log/LogContextScopeListener.java

```diff
@@ -11,8 +11,8 @@
  */
 @Slf4j
 public class LogContextScopeListener {
-  private static final String TRACE_ID_KEY = "dd.trace_id";
-  private static final String SPAN_ID_KEY = "dd.span_id";
+  private static final String TRACE_ID_KEY = "opentelemetry.auto.trace_id";
+  private static final String SPAN_ID_KEY = "opentelemetry.auto.span_id";
```

Comment:
This key will be used in logging config.  I suggest keeping it short.  `ot.trace_id` should be sufficient.

Reply:
👍 I will push this change